prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -192,7 +192,8 @@ class ReportFunction : Logging {
         // add the response to the action table as JSONB and record the httpsStatus
         actionHistory.trackActionResponse(httpResponseMessage, verboseResponse.toString())
         workflowEngine.recordAction(actionHistory)
-        actionHistory.queueMessages(workflowEngine) // Must be done after creating TASK record.
+        actionHistory.queueMessages(workflowEngine)
+        // Must be done after creating TASK record.
```

Comment:
This tweak isn't quite correct - the comment refers specifically to `actionHistory.queueMessages(workflowEngine)`

I admit it could be more clear.    Maybe change it to this, or something like this:

```
// There are two parts to dispatching the next work: (1) entry in TASK table; (2) putting the message in azure queue.
// (2) can only be done once the transaction for (1) is committed.
// Otherwise Azure will occasionally start to process the queued item before the TASK entry is available.
// Hence (2) is done here, rather than in dispatchReport.
actionHistory.queueMessages(workflowEngine)
```

Reply:
Oh, will fix back.